index.html

```diff
@@ -90,6 +87,42 @@ <h3 class="projectname">Project name goes here</h3>
         </div>
     </section>
     <section class="aboutme">
+        <section class="headline">
+            <h1>About</h1>
```

Comment:
- [ ] Your page should have only one `h1` tag which should be the main heading. Kindly change this to an `h2` so as to follow best semantic practices 👍

Reply:
I do this just because those features have the same style on the FIGMA template.

Reply:
Hi @Kingstalux,
I did this just because in the FIGMA template this two features have the same styling but I'll going to changed
Regards

Reply:
Yeah, you can just style the h2 element the same way you styled the h1 element